CONTRIBUTING.md

```diff
@@ -134,6 +134,27 @@ For Jekyll and its dependencies, because we already have the `Gemfile` in the re
 pushd site && bundle install && popd
 ```
 
+### Use  Visual Studio Code Remote - Containers for development
+
+We support development of Vega Lite in VS Code Remote containers.
+The project contains a ".devcontainer/" folder including a Dockerfile which installs all the
+needed development tools inside a Docker container.
+
+See [here](https://code.visualstudio.com/docs/remote/containers) for information on its requirements for different operation systems. (basically a working Docker instalation)
+
+Once you have the [Remote Development extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) installed,
+opening the vega-lite source folder, will propose you to re-open it in a container, which you should confirm.
+
+This will restart VSCode, build the image (could take a while the first time) and then your are inside the container.
+So any terminal shell, is for example, inside the container.
+
+The code is mounted into the container as well automatically.
+
+If you start the site (with `yarn site`) VS code detects the opening of a port and proposes you to open a browser window to view it.
```

Comment:
Please simplify the text using what I have in https://github.com/vega/vega-lite/pull/7809